src/internal/operations/backup.go

```diff
@@ -589,15 +596,21 @@ func (op *BackupOperation) persistResults(
 			opStats.writeErr)
 	}
 
+	op.Results.BytesRead = opStats.k.TotalHashedBytes
+	op.Results.BytesUploaded = opStats.k.TotalUploadedBytes
+	op.Results.ItemsWritten = opStats.k.TotalFileCount
+	op.Results.ResourceOwners = opStats.resourceCount
+
+	if opStats.gc == nil {
+		op.Status = Failed
+		return errors.New("data population never completed")
+	}
+
 	if opStats.readErr == nil && opStats.writeErr == nil && opStats.gc.Successful == 0 {
```

Comment:
I'm just noticing that there is already a check `if opStats.readErr != nil || opStats.writeErr != nil {` on line 589. Don't need to do it in this PR but we can probably remove the extraneous check from this line

Reply:
Same thing that danny pointed out.